ansible/roles/netbox/defaults/main.yml

```diff
@@ -445,6 +445,18 @@ netbox__config_napalm_username: ''
 # This overwrites potentially existing `dynamic configuration settings <https://netbox.readthedocs.io/en/stable/configuration/dynamic-settings/>`__.
 netbox__config_napalm_password: ''
 
+                                                                   # ]]]
+# .. envvar:: netbox__napalm_ssh_generate [[[
+#
+# Generate ``~/.ssh/id_rsa`` SSH private and public key if it doesn't exist.
```

Comment:
That should be `` :file:`~/.ssh/id_rsa` ``? Also, is there any particular reason to not use `ed25519` rather than `RSA` as the key type?

Reply:
I think that `ed25519` might not be supported everywhere - this SSH key will be used on network devices, remember.